### DISCUSSION TITLE
Does skywalking support WebSocket protocol request/response

### DISCUSSION THREAD

**Discussion by whohasthis** [apache/skywalking#8295] [Q&A] [answered]
We would like to use skywalking to monitor our WebSocket protocol service, does skywalking have the ability to monitor WebSocket protocol ?

**Comment by wu-sheng** [comment, marked as answer]
Our document is clear about supported list, https://skywalking.apache.org/docs/skywalking-java/v8.8.0/en/setup/service-agent/java-agent/supported-list/

**Comment by andotorg** [comment]
document supported list missing websocket

**Reply by Superskyyy** [reply]
> document supported list missing websocket

Which plugin specifically?

**Reply by andotorg** [reply]
spring websocket

**Reply by Superskyyy** [reply]
> spring websocket

I do not think it's supported, its not missing.

**Reply by andotorg** [reply]
yes,  I made a mistake, I want to say that he is not supported
